### ISSUE TITLE
Changing separator for CVS-preview

### ISSUE THREAD

**Issue by franiis** [jjuback/gc-excelviewer#72]
Hello, 
I have CSV in following format (first 2 rows):
```
"age";"job";"marital";"education";"default";"housing";"loan";"contact";"month";"day_of_week";"duration";"campaign";"pdays";"previous";"poutcome";"emp.var.rate";"cons.price.idx";"cons.conf.idx";"euribor3m";"nr.employed";"y"
30;"blue-collar";"married";"basic.9y";"no";"yes";"no";"cellular";"may";"fri";487;2;999;0;"nonexistent";-1.8;92.893;-46.2;1.313;5099.1;"no"
```

I want to visualise this data in table (using ExcelViewer), so I create my `settings.json` file (I don't have any other options):
```
{
    "csv-preview.separator": ";"
}
```

After that my preview doesn't show my file (just black view). Earlier (with comma) it showed one cell for whole document. Now even this cell don't appear.

File has about 4000 rows. I restarted VS Code after changing settings and used **CSV: Clear Preview State**.
Am I doing something wrong or there is some problem with extension?

**Comment by jjuback** [comment]
I tried your first two lines with the separator setting and it worked. Maybe one of the 4000 lines has some malformed data, or values that blow up the regular expression parsing. If you upload the file I'll try to figure out what's going on.

**Comment by franiis** [comment]
Hi, thank you for your response. I'm using this data set:
https://archive.ics.uci.edu/ml/machine-learning-databases/00222/
`bank-additional.zip` and smaller file: `bank-additional.csv`.

**Comment by jjuback** [comment]
As a workaround, you can convert your files to have Unix line endings (LF) instead of Windows line endings (CR-LF). I need to fix this in the extension, but the workaround should get you unblocked.

**Comment by franiis** [comment]
Thank you, I didn't thought of that. 
I'll remember that, but fix would be appreciated.

**Comment by jjuback** [comment]
Fixed in version 2.1.25.
